infoblox_client/objects.py

```diff
@@ -779,12 +779,13 @@ def ip(self, ip):
 
 class FixedAddressV4(FixedAddress):
     _infoblox_type = 'fixedaddress'
-    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'network',
-               'options', 'comment']
+    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'name',
+               'network', 'options', 'template', 'comment']
     _search_for_update_fields = ['ipv4addr', 'mac', 'network_view', 'network']
     _all_searchable_fields = _search_for_update_fields
     _shadow_fields = ['_ref', 'ip']
-    _return_fields = ['ipv4addr', 'mac', 'network_view', 'extattrs']
+    _return_fields = ['ipv4addr', 'mac', 'name', 'network', 'network_view',
```

Comment:
As per my earlier comment [https://github.com/infobloxopen/infoblox-client/pull/188#discussion_r171590381](url), I'm not sure `network` belongs to the default set of return fields.